cli/command/bundlefile/bundlefile.go

```diff
@@ -14,15 +14,16 @@ type Bundlefile struct {
 
 // Service is a service from a bundlefile
 type Service struct {
-	Image      string
-	Command    []string          `json:",omitempty"`
-	Args       []string          `json:",omitempty"`
-	Env        []string          `json:",omitempty"`
-	Labels     map[string]string `json:",omitempty"`
-	Ports      []Port            `json:",omitempty"`
-	WorkingDir *string           `json:",omitempty"`
-	User       *string           `json:",omitempty"`
-	Networks   []string          `json:",omitempty"`
+	Image        string
+	Command      []string          `json:",omitempty"`
+	Args         []string          `json:",omitempty"`
+	Env          []string          `json:",omitempty"`
+	Labels       map[string]string `json:",omitempty"`
+	EndpointMode *string           `json:",omitempty"`
```

Comment:
Thanks for your feedback. While, I was wondering if we do not specify `EndpointMode` in `bundlefile.Service`, there must be another way to specify endpoint mode for a service, isn't it?